packages/dialtone-icons/src/keywords-icons.json

```diff
@@ -489,7 +489,23 @@
       ],
       "whisper": [
         "soft"
-      ]
+      ],
+      "power-dialer": [
+        "powerdialer",
+        "dialer",
+        "lightning",
+        "list",
+        "speed"
+      ],
+      "power-dialer-off": [
+        "powerdialer",
+        "dialer",
```

Comment:
Redundant

Reply:
@fede-dp @francisrupert Same as the previous comment just use "powerdialer"